litani

```diff
@@ -288,6 +288,37 @@ def get_args():
         flags = arg.pop("flags")
         transform_jobs_pars.add_argument(*flags, **arg)
 
+    release_html_dir_pars = subs.add_parser("release-html-dir",
+        help="Unlock a previously-locked HTML report directory")
+    release_html_dir_pars.set_defaults(func=lib.litani_report.release_html_dir)
+    for arg in [{
+            "flags": ["-d", "--dir"],
+            "help": "the locked HTML report directory to unlock",
+            "required": True,
+            "type": pathlib.Path,
+            "metavar": "D",
+    }]:
+        flags = arg.pop("flags")
+        release_html_dir_pars.add_argument(*flags, **arg)
+
+    acquire_html_dir_pars = subs.add_parser("acquire-html-dir",
+        help="Print the path to a locked HTML report directory")
+    acquire_html_dir_pars.set_defaults(func=lib.litani_report.acquire_html_dir)
+    for arg in [{
+            "flags": ["-t", "--timeout"],
```

Comment:
This flag:
1. could become required.
2. could remove its default of `0`, so that Litani clients don't issue a command that runs forever.

Reply:
Done, thanks.